v23.1/create-type.md

```diff
@@ -108,11 +131,76 @@ Parameter | Description
 (1 row)
 ~~~
 
+### Create a composite data type
+
+Use the statements below to create a composite data type:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TYPE my_point AS (id INT, x INT, y INT);
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE points (
+    id UUID gen_random_uuid();
+    p mypoint,
+    CONSTRAINT mypoint_xcheck
+        CHECK (((p).x >= 0) AND ((p).x <= 10000)),
+    CONSTRAINT mypoint_ycheck
+        CHECK (((p).y >= 0) AND ((p).y <= 10000))
+);
+~~~
+
+Insert 10,000 randomly generated values of type `mypoint`:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+INSERT INTO points (p) SELECT (floor(random()*10000),floor(random()*10000))::MYPOINT FROM generate_series(1,10000);
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT * FROM points ORDER BY ((p).x) ASC LIMIT 25;
```

Comment:
There are some syntax errors in the example. Here is what it should look like:

```sql
CREATE TYPE my_point AS (id INT, x INT, y INT);

CREATE TABLE points (
    id UUID DEFAULT gen_random_uuid(),
    p my_point,
    CONSTRAINT mypoint_xcheck CHECK (((p).x >= 0) AND ((p).x <= 10000)),
    CONSTRAINT mypoint_ycheck CHECK (((p).y >= 0) AND ((p).y <= 10000))
);

-- Insert 10,000 randomly generated values of type `mypoint`:
INSERT INTO points (p) SELECT (floor(random()*10000),floor(random()*10000), floor(random()*10000))::MY_POINT FROM generate_series(1,10000);
SELECT * FROM points ORDER BY ((p).x) ASC LIMIT 25;
```

Reply:
Fixed the example, thanks for the catch!  Also ran it through a little SQL checker script I've been working on as a side project, so it should be good now. PTAL